examples/src/bin/unbuffered-async-client.rs

```diff
@@ -0,0 +1,242 @@
+use std::error::Error;
+use std::sync::Arc;
+
+use helpers::KB;
+use rustls::client::{ClientConnectionData, UnbufferedClientConnection};
+use rustls::unbuffered::{
+    AppDataRecord, ConnectionState, EncodeError, EncryptError, MayEncryptAppData, UnbufferedStatus,
+};
+#[allow(unused_imports)]
+use rustls::version::{TLS12, TLS13};
+use rustls::{ClientConfig, RootCertStore};
+use rustls_examples as helpers;
+
+#[cfg(not(feature = "async-std"))]
```

Comment:
Import nits: one line per module, no empty lines before the tokio imports, avoid unused imports but added the appropriate feature guard instead.